docs/welcome/index.html

```diff
@@ -8,11 +8,11 @@
   <link rel="icon" href="https://static.hubspot.com/favicon.ico">
 
   <!-- Shepherd themes -->
-  <link rel="stylesheet" href="/shepherd/dist/css/shepherd-theme-arrows.css"/>
+  <link rel="stylesheet" href="../../dist/css/shepherd-theme-arrows.css"/>
 
   <!-- Welcome docs styles -->
-  <link rel="stylesheet" href="/shepherd/docs/welcome/css/prism.css"/>
```

Comment:
Changing these paths will not work. We need these paths for when we deploy to gh-pages.

Reply:
No problem. I just thought it would be easier to understand.